.github/pull_request_template.md

```diff
@@ -2,7 +2,8 @@ What has been done? Why? What problem is being solved?
 
 I didn't forget about (remove if it is not applicable):
 
-- [ ] Well-written commit messages (see [documentation][how-to-write-commit] how to write a commit message)
+- [ ] Well-written commit messages (see [documentation][how-to-write-commit] how
+ to write a commit message)
```

Comment:
You need to double check this after merge. The newline didn't work fine last time.